app/assets/stylesheets/scoreboard/_ongoing-match.scss

```diff
@@ -8,6 +8,25 @@
   );
 }
 
+.ongoing-match-frame {
+  position: relative;
+  width: 100%;
+  height: 95%;
+}
+
+.ongoing-match-next-match {
+  position: absolute;
+  width: 100%;
+  height: 5%;
```

Comment:
without knowing very much about the heirarchy of this, you may benefit from moving to vw + vh instead of percentages and position absolute.

Unless your position absolute, is a child of another positioned element, then just ignore me :)